### ISSUE TITLE
Escape key should close the find/replace modal

### ISSUE THREAD

**Issue by trestletech** [rstudio/rstudio#5115]
On Mac

 - `Cmd+F` to search for text  
 - enter something that doesn't exist in the file
 - Press enter to search

You get a modal titled `Find/Replace` with the subtext `No more occurrences` and the only option is `OK`. My instinct every time I see this is to press escape to get out of the modal which doesn't work. Users have to press enter to close the modal with the keyboard.

(I'm in vim mode on RStudio Desktop, if that matters.)

**Comment by ronblum** [comment]
@trestletech Thank you for raising the issue. I can reproduce this in RStudio Desktop 1.2.1558 on MacOS 10.15 (Catalina). It works correctly in RStudio Server 1.2.1558 on Ubuntu 18.04 reached via Google Chrome on MacOS 10.15. To compare behavior, I tried this out in several applications—Calculator, Skype, Slack, To-Do, and Word—and `Esc` closes the modals in almost all cases. We'll review this as we continue development of RStudio.

Note: This is a regression, as it works in RStudio Desktop 1.1.463.
